### DISCUSSION TITLE
Redirect Bug

### DISCUSSION THREAD

**Discussion by frankl1m** [cefsharp/CefSharp#4251] [General]
Version 105.3.330.0
Using C#

Some sites have a race condition cookies with a single use cookie after that Server Set-Cookie to client and the issue come with redirect 302 if the server say about update the cookie of the request, cefsharp Browser.Load will send same Name Cookie with multiples values at the same time, i mean the old cookie of previous request and the new cookie on the same request and it result on redirect loop all browsers manage it OK. The big cannot be handled while running by user to manually work with any redirect operation for modify the request and cannot get Cookies of any redirect request. Please work on a fix for it

**Comment by amaitland** [comment]
Converted to discussion as [Bug Report](https://github.com/cefsharp/CefSharp/blob/master/.github/ISSUE_TEMPLATE/bug_report.md#bug-report) was not completed.

The behaviour you are describing not something that can be fixed in CefSharp directly. If you can reliably reproduce the issue then you'll need to raise the issue on the CEF issue tracker on bitbucket. CefSharp is just one of many chromium embedded framework (CEF) wrappers. 

https://bitbucket.org/chromiumembedded/cef/issues?status=new&status=open

You can try disabling CEF request interception via the --disable-request-handling-for-testing command line argument. 

> Please work on a fix for it

You'll need to raise the issue on bitbucket. Post a link back here for reference.
